.coderabbit.yml

```
@@ -0,0 +1,20 @@
+# CodeRabbit configuration file
+# https://docs.coderabbit.ai/guides/review-guide
+
+reviews:
+  # Disable PHPMD (PHP Mess Detector) checks
+  tools:
+    phpmd:
+      enabled: false
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)